tools/README.md

```diff
@@ -1 +1,14 @@
-Command line utilities for processing and converting 3D Tiles tilesets.
\ No newline at end of file
+Command line utilities for processing and converting 3D Tiles tilesets.
+
+# Generate B3dm from Glb
+
+Creates a b3dm from a glb with an empty batch table.
+
+## Example
+`node ./bin/generateB3dmFromGlb -i someGlb.glb -o someB3dm.b3dm`
```

Comment:
Maybe tweak this example so it uses files that exist in the specs/data folder. The README in master has some examples.